bentoml/cli/__init__.py

```diff
@@ -144,6 +145,23 @@ def deploy(archive_path, platform, region, stage):
                 'Deploying with "--platform={platform}" is not supported in the current version of BentoML'
                 .format(platform=platform))
 
+    # Example useage: bentoml delete-deployment SERVICE_NAME --platform=aws-lambda
+    @_cli.command()
+    @click.argument('service-name', type=click.STRING)
```

Comment:
I think it makes more sense to use the path to bento archive here, we can load the service name from an archive path. 'service-name' is kinda internal concept and doesn't mean anything to the user.

Reply:
make sense

Reply:
done